pkg/broker/broker.go

```diff
@@ -718,3 +724,35 @@ func (a AnsibleBroker) AddSpec(spec apb.Spec) (*CatalogResponse, error) {
 	service := SpecToService(&spec)
 	return &CatalogResponse{Services: []Service{service}}, nil
 }
+
+func (a AnsibleBroker) RemoveSpec(specID string) error {
+	spec, err := a.dao.GetSpec(specID)
+	if client.IsKeyNotFound(err) {
+		return ErrorNotFound
+	}
```

Comment:
Is this really an error case? Removing a non existent spec seems like a noop instead of an error to me.